test/src/core/util/talentIO.spec.js

```diff
@@ -0,0 +1,143 @@
+/*****************************************************************************
+ * Copyright (c) 2021 Bosch.IO GmbH
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ ****************************************************************************/
+
+const { DEFAULT_TYPE, DEFAULT_INSTANCE } = require('../../../../src/core/constants');
+const {
+    TalentInput,
+    TalentOutput,
+    FeatureMetadata
+} = require('../../../../src/core/util/talentIO');
+
+describe('core.util.talentIO', () => {
+    const ev = require('../../../resources/talentIO.event.json');
+
+    describe('TalentInput', () => {
```

Comment:
These seems to only tests for good cases but no error case (e.g. invalid / broken json)

Reply:
Since the json comes from our platform, it's ensured, that only well formatted JSON will be taken. Since jsonModel and jsonQuery are used to retrieve all the stuff, which have built-in error handling, it's currently not necessary to test the "unhappy" path. We can add this in the future.

Reply:
I agree with you that the platform has to provide a correct json but I see TalentIO still dedicated from it and TalentIO (as a central point) should not result in an undefined behaviour if something gets broken. I think the point is now how the TalentIO will handle such cases... I assume that it is already throwing an error/exception (Which could be tested here) or it is something which has to be added as a feature later on.

Reply:
> I assume that it is already throwing an error/exception (Which could be tested here) or it is something which has to be added as a feature later on.

Yes, it will throw an exception, if anything goes wrong

Reply:
Ah sorry I was not aware that jsonModel & jsonQuery is part of TalentIO and are repsonsible for that (and has been tested there already)